scp.1

```diff
@@ -38,9 +37,21 @@ It uses
 .Xr ssh 1
 for data transfer, and uses the same authentication and provides the
 same security as a login session.
-The scp protocol requires execution of the remote user's shell to perform
+Since OpenSSH 8.7,
+.Nm
+has used the SFTP protocol for file transfers and requires a working
+.Xr sftp-server 8
+(or equivalent) to be present on the server.
+Use of the legacy SCP protocol remains available via the
+.Fl O
```

Comment:
Is this option missing in https://github.com/djmdjm/openssh-wip/pull/7/files#diff-b6c2513bd54d1f9dc161c39485138365b5d1f21f38f44e825e096bf8a911c4d2R21 an oversight or was it on purpose?

Reply:
good catch. -O is missing from the SYNOPSIS section. I'll add it in the next patchset